Workiva/ixbrl-viewer
# XT-2861 XT-2862 Document Summary Pane

#### Reason for change
Request to add document summary information to viewer.

Potential follow ups:
* hovering over namespace prefixes displays a tooltip containing the full namespace.
* file summary provides download links.

#### Description of change
<img width="499" alt="Screenshot 2023-05-09 at 10 16 44 PM" src="https://github.com/Workiva/ixbrl-viewer/assets/46454775/0b266661-ff0a-4ce3-a3fc-813e47f52500">

#### Steps to Test
* Generate a viewer and confirm the details in the summary pane are correct.
* Verify that the summary details pane is usable within the inspector at a variety of widths.

**review**:
@Workiva/xt
@paulwarren-wk
